bg-geocode/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG  DOCKER_REPOSITORY
+ARG  MANAGE_COURSES_BACKEND_IMAGE
```

Comment:
Should this be `MANAGE_COURSE_BG_GEOCODING_IMAGE`? Or if this really is a generic name, `DOCKER_HUB_IMAGE_NAME`?

Reply:
No `MANAGE_COURSES_BACKEND_IMAGE` is a correct variable name. GeoCoding image will be built from mc_backend_image